src/data/wallets/wallet-data.ts

```diff
@@ -1859,6 +1860,66 @@ export const walletsData: WalletData[] = [
     documentation: "https://docs.shapeshift.com/",
     new_to_crypto: true,
   },
+  {
+    last_updated: "2024-06-20",
+    name: "Gem Wallet",
+    image: GemWalletImage,
+    brand_color: "#2D5BE6",
+    url: "https://gemwallet.com/",
+    active_development_team: true,
+    languages_supported: [
+      "en",
+      "es",
+      "de",
+      "fr",
+      "zh",
+      "ja",
+      "pt",
+      "ru",
+      "tr",
+      "uk",
+      "ko",
+      "ar",
+      "ua",
+      "vi",
+      "pl",
+    ],
+    twitter: "https://twitter.com/GemWalletApp",
+    discord: "https://discord.gg/4jpxtwT8r6",
+    reddit: "",
+    telegram: "https://t.me/gemwallet",
+    ios: true,
+    android: true,
+    linux: false,
+    windows: false,
+    macOS: true,
+    firefox: false,
+    chromium: false,
+    hardware: false,
+    open_source: true,
+    repo_url: "https://github.com/gemwalletcom/",
+    non_custodial: true,
+    security_audit: [],
+    scam_protection: false,
+    hardware_support: false,
+    rpc_importing: false,
+    nft_support: false,
+    connect_to_dapps: true,
+    staking: true,
+    swaps: true,
+    multichain: true,
+    layer_2: true,
+    gas_fee_customization: false,
+    ens_support: true,
+    erc_20_support: true,
+    buy_crypto: true,
+    withdraw_crypto: false,
+    multisig: false,
+    social_recovery: false,
+    onboard_documentation: "https://docs.gemwallet.com/",
+    documentation: "https://docs.gemwallet.com/",
+    new_to_crypto: true,
```

Comment:
The wallets were recently updated wth a curated list for "new to crypto" to improve UX for beginners after much research. Let's not add this flag just yet; can be considered for the future (cc: @konopkja)

```suggestion
```

Reply:
remove the tag